examples/persistence_transform.py

```diff
@@ -0,0 +1,52 @@
+"""
+========================================
+Persistence Transform in MapSequence
+========================================
+Persistence Transform is a simple image processing technique that is useful for the visualization
+and depiction of gradually evolving structures.
+This example illustrates persistence transform using the AIA_193_CUTOUT series.
+
+.. note::
+   This example requires ``mpl_animators`` to be installed.
```

Comment:
```suggestion
.. note::

   This example requires `mpl_animators` to be installed.
```
If this link does not work, we should add it to the config.